### ISSUE TITLE
Problem parsing dependency 21 of keybase:amd64=4.6.0-20191010154240.134c2d892b

### ISSUE THREAD

**Issue by alancwoo** [keybase/client#20482]
I'm encountering this error when installing on Ubuntu 19.10:

`sudo apt install ./keybase_amd64.deb`

> Reading package lists... Error!
> E: Problem parsing dependency 21 of keybase:amd64=4.6.0-20191010154240.134c2d892b
> E: Error occurred while processing keybase (NewVersion2)
> E: Problem with MergeList /home/awoo/keybase_amd64.deb
> E: The package lists or status file could not be parsed or opened.

**Comment by Alasano** [comment]
Same here - Fresh Pop!_OS 19.10 install

**Comment by connesy** [comment]
Same here, fresh Ubuntu 19.10 install.

**Comment by llimllib** [comment]
I've experienced this both on a fresh ubuntu 19.10 and ubuntu mate 19.10

**Comment by alancwoo** [comment]
I was finally able to install by manually installing dependencies and using dpkg instead

> sudo apt-get install --fix-broken
> sudo apt-get install lib-gconf-2-4
> sudo dpkg -i ./keybase_amd64.deb
> run_keybase

**Comment by connesy** [comment]
After `sudo dpkg -i ./keybase_amd64.deb` I got the message 
```
$ sudo dpkg -i ./keybase_amd64.deb
Selecting previously unselected package keybase.
(Reading database ... 207146 files and directories currently installed.)
Preparing to unpack ./keybase_amd64.deb ...
Unpacking keybase (4.7.0-20191021162721.67d9526320) ...
dpkg: dependency problems prevent configuration of keybase:
 keybase depends on libappindicator1; however:
  Package libappindicator1 is not installed.
```
I then ran
```
sudo apt-get -f install
sudo dpkg -i ./keybase_amd64.deb
run_keybase
```
and everything worked.

**Comment by heronhaye** [comment]
Should be fixed now with Keybase 4.7.1. Please try it out!
